openedx/core/djangoapps/credentials/models.py

```diff
@@ -23,6 +23,9 @@ class Meta(object):
     CACHE_KEY = 'credentials.api.data'
 
     internal_service_url = models.URLField(verbose_name=_("Internal Service URL"))
+    version = models.PositiveIntegerField(
```

Comment:
Would that be better? 
I just did it this way because the config file in program/models combines the url and version